packages/create-vite/template-qwik-ts/src/app.tsx

```diff
@@ -0,0 +1,44 @@
+import { component$, useSignal } from '@builder.io/qwik'
+
+import qwikLogo from './assets/qwik.svg'
+import viteLogo from '/vite.svg'
+import './app.css'
+
+export const App = component$(() => {
+  const count = useSignal(0)
+
+  return (
+    <>
+      <div>
+        <a href="https://vitejs.dev" target="_blank">
+          <img
+            src={viteLogo}
+            width={32}
+            height={32}
```

Comment:
The Vite logo has the same size as the framework logo in other templates. See 
<img width="786" alt="image" src="https://github.com/vitejs/vite/assets/583075/430f1e71-a992-48fd-8498-d1e25028c049">

Same for the JS template